layers/icmp6.go

```diff
@@ -31,6 +29,47 @@ const (
 	ICMPv6TypeRedirect              = 137
 )
 
+var (
+	icmpv6TypeStringMap = map[uint8]string{
+		1:   "DestinationUnreachable",
+		2:   "PacketTooBig",
+		3:   "TimeExceeded",
+		4:   "ParameterProblem",
+		128: "EchoRequest",
+		129: "EchoReply",
+		133: "RouterSolicitation",
+		134: "RouterAdvertisement",
+		135: "NeighborSolicitation",
+		136: "NeighborAdvertisement",
+		137: "Redirect",
+	}
+
+	icmpv6CodeStringMap = map[[2]uint8]string{
+		[2]uint8{1, 0}:   "NoRouteToDst",
+		[2]uint8{1, 1}:   "AdminProhibited",
+		[2]uint8{1, 2}:   "BeyondScopeOfSrc",
+		[2]uint8{1, 3}:   "AddressUnreachable",
+		[2]uint8{1, 4}:   "PortUnreachable",
+		[2]uint8{1, 5}:   "SrcAddressFailedPolicy",
+		[2]uint8{1, 6}:   "RejectRouteToDst",
+		[2]uint8{2, 0}:   "",
```

Comment:
you shouldn't need to add empty strings anywhere... a map will return an empty string if the key doesn't exist.